sourcecode/hub/resources/views/content/standalone-view.blade.php

```diff
@@ -1,71 +1,64 @@
 <x-layout>
     <x-slot name="title">{{ trans('messages.standalone-view') }}</x-slot>
-
     <div class="container">
         <div class="row justify-content-center">
             <div class="row justify-content-center">
                 <x-lti-launch :launch="$launch" />
             </div>
         </div>
-
         <div class="row justify-content-center mt-4">
-            <div class="">
-                <div class="buttons-container d-flex justify-content-center">
-                    <button type="button" class="btn btn-secondary mx-2" data-bs-toggle="modal" data-bs-target="#reportModal">
-                        {{ trans('messages.report-content')}}
-                    </button>
-
-                    <button type="button" class="btn btn-primary mx-2">
-                        {{ trans('messages.more-details')}}
-                    </button>
-                </div>
+            <div class="buttons-container d-flex justify-content-center">
+                <button type="button" class="btn btn-secondary mx-2" data-bs-toggle="modal" data-bs-target="#reportModal" aria-label="{{ trans('messages.report-content')}}">
+                    {{ trans('messages.report-content')}}
+                </button>
+                <button type="button" class="btn btn-primary mx-2" aria-label="{{ trans('messages.more-details')}}">
```

Comment:
The content of the element is a readable text, no need to repeat it using `aria-label`